src/scala/com/github/johnynek/bazel_deps/Decoders.scala

```diff
@@ -15,6 +15,12 @@ object Decoders {
       case "runtime_deps" => Right(Transitivity.RuntimeDeps)
       case other => Left(s"unrecogized transitivity: $other")
     }
+  implicit val resolverCacheDecoder: Decoder[ResolverCache] =
+    Decoder.decodeString.emap {
+      case "local" => Right(ResolverCache.Local)
+      case "bazel_output_base" => Right(ResolverCache.BazelOutputBase)
```

Comment:
I can see the benefit of a fixed location.  I think it would work with a "path:" sentinel that is stripped off.  Or use a more structured format in the yaml, e.g.:

```
options:
 resolverCache:
  computed: "local" # or "bazel_output_path"
```
or
```
options:
 resolverCache:
  path: "/some/path"
```

where one of either `computed` or `path` would be required if `resolverCache` is specified.